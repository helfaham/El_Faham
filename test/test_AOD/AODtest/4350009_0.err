WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
Cloning into 'HaNaMiniAnalyzer'...
Already on 'AOD18'
/pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/BTagWeight.cc: In member function 'float BTagWeight::TagScaleFactor(pat::Jet, bool)':
/pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/BTagWeight.cc:118:8: warning: unused variable 'MinJetPt' [-Wunused-variable]
  float MinJetPt = 20.;
        ^~~~~~~~
/pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/BTagWeight.cc:119:8: warning: unused variable 'MaxBJetPt' [-Wunused-variable]
  float MaxBJetPt = 670., MaxLJetPt = 1000.;
        ^~~~~~~~~
/pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/BTagWeight.cc:119:26: warning: unused variable 'MaxLJetPt' [-Wunused-variable]
  float MaxBJetPt = 670., MaxLJetPt = 1000.;
                          ^~~~~~~~~
/pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/BTagWeight.cc:120:15: warning: unused variable 'JetPt' [-Wunused-variable]
         float JetPt = jet.pt(); bool DoubleUncertainty = false;
               ^~~~~
/pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/BTagWeight.cc:120:38: warning: unused variable 'DoubleUncertainty' [-Wunused-variable]
         float JetPt = jet.pt(); bool DoubleUncertainty = false;
                                      ^~~~~~~~~~~~~~~~~
/pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/BTagWeight.cc:121:6: warning: unused variable 'flavour' [-Wunused-variable]
  int flavour = fabs(jet.hadronFlavour());
      ^~~~~~~
/bin/sh: line 1:   949 Segmentation fault      edmWriteConfigs -p /pool/condor/dir_23299/CMSSW_9_4_7/tmp/slc6_amd64_gcc630/src/Haamm/HaNaMiniAnalyzer/plugins/HaammHaNaMiniAnalyzerAuto/libHaammHaNaMiniAnalyzerAuto.so
gmake: *** [lib/slc6_amd64_gcc630/HaammHaNaMiniAnalyzerAuto.edmplugin] Error 1
condor_exec.exe: line 34:   953 Segmentation fault      cmsRun PUAnalyzer_cfg.py sample=$5 job=$FILEID output=$6 maxEvents=-1 nFilesPerJob=$8
ls: cannot access out*AODtest*.root: No such file or directory
ls: cannot access *out*AODtest*.root: No such file or directory
mkdir: cannot create directory '/./helfaham/': Permission denied
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
cp: missing destination file operand after '/./helfaham//'
Try 'cp --help' for more information.
