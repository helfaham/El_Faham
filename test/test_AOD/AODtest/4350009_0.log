000 (4350009.000.000) 05/01 16:04:08 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_2288_f4c7_91>
...
040 (4350009.000.000) 05/01 16:21:36 Started transferring input files
	Transferring to host: <188.184.211.30:9618?addrs=188.184.211.30-9618+[2001-1458-301-10b--100-103]-9618&noUDP&sock=18942_1a3b_252976>
...
040 (4350009.000.000) 05/01 16:21:36 Finished transferring input files
...
001 (4350009.000.000) 05/01 16:21:37 Job executing on host: <188.184.211.30:9618?addrs=188.184.211.30-9618+[2001-1458-301-10b--100-103]-9618&noUDP&sock=18808_4d56_3>
...
006 (4350009.000.000) 05/01 16:21:46 Image size of job updated: 38820
	35  -  MemoryUsage of job (MB)
	35732  -  ResidentSetSize of job (KB)
...
006 (4350009.000.000) 05/01 16:26:38 Image size of job updated: 171796
	36  -  MemoryUsage of job (MB)
	36532  -  ResidentSetSize of job (KB)
...
040 (4350009.000.000) 05/01 16:26:38 Started transferring output files
...
040 (4350009.000.000) 05/01 16:26:38 Finished transferring output files
...
005 (4350009.000.000) 05/01 16:26:38 Job terminated.
	(1) Normal termination (return value 1)
		Usr 0 00:01:08, Sys 0 00:00:10  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:01:08, Sys 0 00:00:10  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	8690  -  Run Bytes Sent By Job
	3580  -  Run Bytes Received By Job
	8690  -  Total Bytes Sent By Job
	3580  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :   101431        4   2978456 
	   Memory (MB)          :       36     2000      2000 
...
