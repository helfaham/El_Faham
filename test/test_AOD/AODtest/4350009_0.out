subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=helfaham/CN=834699/CN=Hesham El Faham/CN=890439362
issuer    : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=helfaham/CN=834699/CN=Hesham El Faham
identity  : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=helfaham/CN=834699/CN=Hesham El Faham
type      : RFC3820 compliant impersonation proxy
strength  : 2048
path      : /afs/cern.ch/user/h/helfaham/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/test/test_AOD/.x509up_u119445
timeleft  : 999:42:23
key usage : Digital Signature, Key Encipherment
WARNING: Developer's area is created for architecture slc6_amd64_gcc630 while your current OS is SLC7.
Your branch is up-to-date with 'origin/AOD18'.
SCRAM warning: You are trying to compile/build for architecture slc6_amd64_gcc630 on SLC7 OS which might not work.
If you know this SCRAM_ARCH/OS combination works then please first run 'scram build --ignore-arch'.
>> Local Products Rules ..... started
>> Local Products Rules ..... done
>> Entering Package Haamm/HaNaMiniAnalyzer
>> Creating project symlinks
Entering library rule at Haamm/HaNaMiniAnalyzer
>> Compiling  /pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/JetReader.cc 
>> Compiling  /pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/PackedCandidateReader.cc 
>> Compiling  /pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/DiMuonReader.cc 
>> Compiling  /pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/VertexReader.cc 
>> Compiling  /pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/LHEEventReader.cc 
>> Compiling  /pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/GenEventInfoProductReader.cc 
>> Compiling  /pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/METReader.cc 
>> Compiling  /pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/Histograms.cc 
>> Compiling  /pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/Category.cc 
>> Compiling  /pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/TtGenEvent.cc 
>> Compiling  /pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/HLTReader.cc 
>> Compiling  /pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/BTagWeight.cc 
>> Compiling  /pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/src/HaNaBaseMiniAnalyzer.cc 
>> Building shared library tmp/slc6_amd64_gcc630/src/Haamm/HaNaMiniAnalyzer/src/HaammHaNaMiniAnalyzer/libHaammHaNaMiniAnalyzer.so
Copying tmp/slc6_amd64_gcc630/src/Haamm/HaNaMiniAnalyzer/src/HaammHaNaMiniAnalyzer/libHaammHaNaMiniAnalyzer.so to productstore area:
Leaving library rule at Haamm/HaNaMiniAnalyzer
Entering library rule at src/Haamm/HaNaMiniAnalyzer/plugins
>> Compiling edm plugin /pool/condor/dir_23299/CMSSW_9_4_7/src/Haamm/HaNaMiniAnalyzer/plugins/PUStudies.cc 
>> Building edm plugin tmp/slc6_amd64_gcc630/src/Haamm/HaNaMiniAnalyzer/plugins/HaammHaNaMiniAnalyzerAuto/libHaammHaNaMiniAnalyzerAuto.so
Leaving library rule at src/Haamm/HaNaMiniAnalyzer/plugins
@@@@ Running edmWriteConfigs for HaammHaNaMiniAnalyzerAuto
config/SCRAM/GMake/Makefile.rules:2068: recipe for target 'lib/slc6_amd64_gcc630/HaammHaNaMiniAnalyzerAuto.edmplugin' failed
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
0
cmsRun PUAnalyzer_cfg.py sample=AODtest job=0 output=out maxEvents=-1 nFilesPerJob=2
total 72
-rw-r--r--. 1 helfaham zh 13937 May  1 16:21 Cert_314472-325175_13TeV_PromptReco_Collisions18_JSON.txt
-rw-r--r--. 1 helfaham zh  2487 May  1 16:21 FinalDraw.py
-rw-r--r--. 1 helfaham zh  7220 May  1 16:21 PUAnalyzer_cfg.py
drwxr-xr-x. 2 helfaham zh  4096 May  1 16:21 PUStudies
-rw-r--r--. 1 helfaham zh  6245 May  1 16:21 Plotter.py
drwxr-xr-x. 2 helfaham zh  4096 May  1 16:21 SamplesPU
-rwxr-xr-x. 1 helfaham zh  3580 May  1 16:21 SetupAndRun.sh
-rw-r--r--. 1 helfaham zh   565 May  1 16:21 SubmitPlotter.py
-rwxr-xr-x. 1 helfaham zh  2627 May  1 16:21 condor.py
-rwxr-xr-x. 1 helfaham zh  1370 May  1 16:21 hadd.py
-rw-r--r--. 1 helfaham zh  4574 May  1 16:21 lxbatch.py
-rw-r--r--. 1 helfaham zh   220 May  1 16:21 runPlotter.sh
1th Try
2th Try
3th Try
4th Try
5th Try
6th Try
7th Try
8th Try
9th Try
10th Try
11th Try
12th Try
13th Try
14th Try
15th Try
16th Try
17th Try
18th Try
19th Try
20th Try
21th Try
The file was not copied to destination after 20 tries
